plugin/go.vim

```diff
@@ -4,6 +4,29 @@ if exists("g:go_loaded_install")
 endif
 let g:go_loaded_install = 1
 
+" Not using the has('patch-7.4.1689') syntax because that wasn't added until
+" 7.4.237, and we want to be sure this works for everyone (this is also why
+" we're not using utils#EchoError()).
+"
+" Version 7.4.1689 was chosen because that's what the most recent Ubuntu LTS
+" release (16.04) uses.
+if
+      \ !exists('g:go_no_version_warning') &&
```

Comment:
I wouldn't be surprised is someone sets this to `0` to try to re-enable the warning. Maybe just check that it's not `1`?